hugo/content/contact.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Contact
+toc: true
+---
+
+# Contact
+
+You can contact us in several ways:
+
+## Community Channels
+
+* Our [issue tracker on GitHub](https://github.com/mumble-voip/mumble/issues)
+* [Discussions on GitHub](https://github.com/mumble-voip/mumble/discussions)
+* IRC: [#mumble on irc.freenode.net](irc://irc.freenode.org/mumble)
+* Chat: [Mumble room on Riot/Matrix](https://riot.im/app/#/room/#mumble:matrix.org) (shares messages with IRC)
+* Our [Forums](https://forums.mumble.info/)
+
+## Email
+
+**General contact:**
+
+contact [at] mumble [dot] info
```

Comment:
Optional: Do you have an encryption key?

Reply:
For email? I don't think so

Reply:
Yes. Ok.